### DISCUSSION TITLE
ar_flutter_plugin and firebase messaging conflict together in iOS

### DISCUSSION THREAD

**Discussion by naved997** [firebase/flutterfire#13279] [Q&A]
I am creating a flutter app based on Android and iOS in which i want to send notification through app and it works on Android but in iOS it gives error:
[!] CocoaPods could not find compatible versions for pod "nanopb":
      In Podfile:
        ar_flutter_plugin (from `.symlinks/plugins/ar_flutter_plugin/ios`) was resolved to 0.6.2, which depends on
          ARCore/CloudAnchors (~> 1.32.0) was resolved to 1.32.0, which depends on
            nanopb (< 2.30910.0, >= 2.30908.0)

        firebase_messaging (from `.symlinks/plugins/firebase_messaging/ios`) was resolved to 15.1.0, which depends on
          Firebase/Messaging (= 11.0.0) was resolved to 11.0.0, which depends on
            FirebaseMessaging (~> 11.0.0) was resolved to 11.0.0, which depends on
              nanopb (~> 3.30910.0)
              
  Any solution for this?

**Comment by NikolaiEcker** [comment]
Hey 
If anybody found a fix for this issue can you pls share? I'm realy desperate for a fix...
